lib/ingestors/concerns/sitemap_helpers.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module Ingestors
+  module Concerns
+    # From a sitemap.{xml|txt} or a single URL, get the list of URLs (= sources)
+    module SitemapHelpers
+      private
+
+      # Reads either a sitemap.{xml|txt} or a single URL
+      # Returns a list of URLs from 1 to n URLs
+      def get_sources(source_url)
```

Comment:
Can we rename this method to `parse_sitemap`?

Just so it doesn't get confused with TeSS' concept of "sources"